src/app/static/src/tests/filteredData/getters.test.ts

```diff
@@ -137,7 +132,7 @@ describe("FilteredData getters", () => {
         const testFilters = {
             age: [{id: "age1", name: "0-4"}, {id: "age2", name: "5-9"}],
             quarter: [{id: "1", name: "2019 Q1"}],
-            indicator: []
+            indicators: []
```

Comment:
We probably ought to decide on a plural/singular policy for filter options, and stick to it! If it's 'indicators' then shouldn't it also be 'quarters' and 'ages'...?